### PR TITLE
feature: PWA support

### LINKED ISSUE #16: Add PWA support to site
### Description

Fully add and implement PWA functionality for the website

#### Notes

- [x] add `gatsby-plugin-offline`
- [x] configure `gatsby-plugin-manifest`

### PR DESCRIPTION
**Pull request by christoff-linde**
### Notes
- Add PWA support
- Add SEO data

### CODE REVIEW

File: gatsby-config.ts [hunk 1]
```diff
@@ -37,11 +33,21 @@ const config: GatsbyConfig = {
     "gatsby-plugin-sass",
     "gatsby-plugin-use-query-params",
     {
-      resolve: `gatsby-plugin-offline`,
+      resolve: "gatsby-plugin-manifest",
```
Comment (ksamson2) [inline gatsby-config.ts:36]: What does this change mean?

Comment (ksamson2) [reply gatsby-config.ts:36]: Ahh I see, for anyone else who might be confused, take a look here: 
https://www.gatsbyjs.com/plugins/gatsby-plugin-manifest/

File: src/components/Seo/index.tsx [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+        {
+          property: `og:description`,
+          content: metaDescription,
+        },
+        {
+          property: `og:type`,
+          content: `website`,
+        },
+        {
+          name: `twitter:card`,
```
Comment (ksamson2) [inline src/components/Seo/index.tsx:87]: Small question, why do we need this property?

**Review by ksamson2 — DISMISSED** [review]

File: gatsby-config.ts (hunk 1 above)
Comment (christoff-linde) [reply gatsby-config.ts:36]: Yeah, literally just moved stuff around 😄

**Pull request merged**